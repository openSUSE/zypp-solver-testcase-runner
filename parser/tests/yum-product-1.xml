<?xml version="1.0" encoding="UTF-8"?>
<product 
    xmlns="http://novell.com/package/metadata/suse/product"
    xmlns:product="http://novell.com/package/metadata/suse/product"
    xmlns:yum="http://linux.duke.edu/metadata/common" 
    xmlns:rpm="http://linux.duke.edu/metadata/rpm" 
    xmlns:suse="http://novell.com/package/metadata/suse/common"
    type="add-on">
  <vendor>Novell Inc.</vendor>
  <name>OES</name>
  <version epoch="0" ver="1" rel="2"/>
  <displayname lang="en">Open Enterprise Server</displayname>
  <description lang="en">Opens your server to enterprise</description>
  <rpm:provides><rpm:entry kind="product" name="OES"/></rpm:provides>
</product>

