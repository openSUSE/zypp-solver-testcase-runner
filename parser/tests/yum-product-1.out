-------------------------------------------------
Product Data: 
  type: add-on
  vendor: Novell Inc.
  name: OES
  display name: [en] Open Enterprise Server
  description: [en] Opens your server to enterprise
  epoch: 0
  version: 1
  release: 2
  provides: [product] OES  -
  conflicts: 
  obsoletes: 
  freshen: 
  requires: 
Exit Code: 0
